src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -367,6 +376,13 @@ export default defineComponent({
       }
     },
 
+    displayDuration: function () {
+      if (this.useDeArrowTitles && (this.duration === '' || this.duration === '0:00') && this.deArrowCache?.videoDuration) {
```

Comment:
How to test this

Reply:
1. In the "Subscription Settings" turn on "Fetch Feeds from RSS".
2. Subscribe to a channel that has dearrow titles e.g. https://www.youtube.com/@MrBeast
3. On the subscription page any unwatched videos from that channel should now have durations shown (for watched videos the duration comes from the watch history).